pkg/cloud/services/secretsmanager/cloudinit.go

```diff
@@ -65,8 +66,9 @@ func GenerateCloudInitMIMEDocument(secretID, region string) ([]byte, error) {
 	}
 
 	scriptVariables := scriptVariables{
-		SecretID: secretID,
-		Region:   region,
+		SecretPrefix: secretPrefix,
+		Chunks:       chunks - 1,
```

Comment:
this is a cheat for avoiding bash, shouldn't do this.

Reply:
undo, then in bash, we can do `CHUNK_SEQUENCE_END=$((CHUNKS-1))`